ui/static/ui/js/listing.js

```diff
@@ -1,6 +1,78 @@
 define(['jquery', 'bootstrap', 'icheck', 'csrf'], function(jQuery) {
   'use strict';
 
+  /* global $ */
+
+  function formatGroupName(string) {
+    string = string.charAt(0).toUpperCase() + string.slice(1);
+    return string.substring(0, string.length - 1);
+  }
+
+  /* This is going to grow up to check whether
+   * the name is a valid Kerberos account
+   */
+  function isMITEmail(kerberosId) {
+    var email = kerberosId + '@mit.edu';
```

Comment:
sorry, this comment is totally on the wrong line of code, should have been: https://github.com/mitodl/lore/pull/271/files#diff-5880d9dfd3a8657f9866e255341629a9R27